packages/desktop-client/src/components/LoggedInUser.tsx

```diff
@@ -205,9 +205,7 @@ export function LoggedInUser({
         userData?.displayName &&
         !hasSyncedPrefs && (
           <small>
-            <Trans>
-              (logged in as: <span>{userData?.displayName}</span>)
-            </Trans>
+            (<Trans>logged in as:</Trans> <span>{userData?.displayName}</span>)
```

Comment:
Here and a few other places we're inconsistent about whether the thing following the ':' is part of the string as a variable or not. My thinking is we should include it where possible, any thoughts?

Reply:
I agree, it would be better always included. I was having the same problem as above here (object literal in a child of the Trans element) and didn't use the nested Trans. If I can work out a better solution we should definitely include where possible